### ISSUE TITLE
Snap on Ubuntu not working but flatpak on same does...

### ISSUE THREAD

**Issue by no-akira** [SeaDve/Mousai#122]
**Affected ve

- io.github.seadve.Mousai 0.7.6
- Flatpak: false
- Language: en_GB, en, en, C
- Distribution: Ubuntu Core 22
- Desktop Session: ubuntu
- Display Server: x11

- GTK 4.12.2
- Libadwaita 1.4.0
- Libsoup 3.4.3
- GStreamer 1.20.3
- Libpulse 15.99.0rsion**

flatpak on same system debug screen

- io.github.seadve.Mousai 0.7.6
- Flatpak: true
- Language: en_GB, en, en, C
- Distribution: GNOME 45 (Flatpak runtime)
- Desktop Session: ubuntu
- Display Server: x11

- GTK 4.12.4
- Libadwaita 1.4.2
- Libsoup 3.4.4
- GStreamer 1.22.8
- Libpulse 17.0.0

**Bug summary**
Basically flatpak works fine successfully identifies song from Desktop Audio

Snap from snapcraft.io doesn't work on Ubuntu 22.04 LTS which is a constant theme...

**Comment by SeaDve** [comment]
Thanks for taking time to report the issue. However, without the necessary information, it is difficult to address the issue. Please fill up ~all~ the information from the issue template (e.g., how to reproduce it, the logs). Feel free to reopen the issue once the information are filled in.

**Comment by SeaDve** [comment]
cc @soumyaDghosh

**Comment by SeaDve** [comment]
Possible duplicate of #111 #108 #103
